src/microbe_stage/editor/MicrobeEditorPatchMap.tscn

```diff
@@ -108,8 +108,6 @@ margin_right = 1500.0
 margin_bottom = 1500.0
 rect_min_size = Vector2( 1500, 1500 )
 DrawDefaultMapIfEmpty = false
-PatchNodeWidth = 64.0
```

Comment:
Unintended changes?

Reply:
I think I deleted this export value and moved it to Constants.cs in my last few prs so this is automatically deleted.

Surely this is unintended, but this is the way (

Reply:
I suppose it is, I can't find a variable named PatchNodeWidth anymore.